### PR TITLE
Create blank pages and add header

### PR DESCRIPTION
**Pull request by Tkach-Illia**

### CODE REVIEW

**Review by azazak123 — COMMENTED** [review]
In general good. However, could you please explain why we should use CSR instead of SSR?

**Comment by Tkach-Illia** [comment]
> In general good. However, could you please explain why we should use CSR instead of SSR?

Because we must have a lot of dynamic content, such as chat.

**Review by azazak123 — APPROVED** [review]
LGTM!

File: front-end/src/router/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import Map from "../pages/Map";
+import Lost from "../pages/Lost";
+import AccountIdPage from "../pages/AccountIdPage";
+import PostIdPage from "../pages/PostIdPage";
+
+export const routes = [
+    {path: '/posts_find', element: <Main/>},
+    {path: '/map', element: <Map/>},
+    {path: '/posts_lost', element: <Lost/>}
+    // {path: '/posts/:id', element: <PostIdPage/>},
```
Comment (azazak123) [inline front-end/src/router/index.js]: Let's remove comments

**Review by azazak123 — COMMENTED** [review]
One comment and LGTM

**Pull request merged**